packages/patternfly-4/src/components/_core/Documentation/index.js

```diff
@@ -58,6 +58,11 @@ export default class Documentation extends React.Component {
                 <PageSection variant={PageSectionVariants.light} className="pf-site-background-medium">
                   <AutoLinkHeader size="md" is="h1" className="pf4-site-framework-title">HTML</AutoLinkHeader>
                   <AutoLinkHeader size="4xl" is="h2" suffix="-title" className="pf-u-mt-sm pf-u-mb-md">{heading}</AutoLinkHeader>
+                  {location.pathname.indexOf('experimental') >= 0 &&
+                    <Alert isInline variant="info" title="Experimental feature" style={{ width: '800px', marginBottom: 'var(--pf-global--spacer--md)' }}>
```

Comment:
@redallen can you remove the width attribute all together

Reply:
Done.